changelogs/fragments/nxos_bugfixes.yaml

```diff
@@ -29,3 +29,6 @@ bugfixes:
 - nxos_snmp_traps - Fix nxos_snmp_traps issues (https://github.com/ansible/ansible/pull/39444)
 - nxos_linkagg - nxos_linkagg abbreviated form issue (https://github.com/ansible/ansible/pull/39591)
 - nxos_snmp_user - Fix nxos_snmp_user (https://github.com/ansible/ansible/pull/39760)
+- nxos_logging - remove purge from nxos_logging doc, argspec (https://github.com/ansible/ansible/pull/39947)
```

Comment:
@trishnaguha Adding items to existing changelog fragments will cause the new changelog items to show up in whatever release the fragment was originally added in, which I'm guessing was *not* your intent. It's OK to do that between releases, but let's say this fragment was added in 2.5.1- your changes will now show up under 2.5.1 instead of 2.5.3 (which is incorrect).

Reply:
@nitzmahone ah I wasn't aware of that. I will open a separate PR to fix that. Thanks.

Reply:
https://github.com/ansible/ansible/pull/40263